app/views/maintenances/index.html.haml

```diff
@@ -1,4 +1,18 @@
+=link_to t('.add_new'), new_maintenance_path(@maintenance), class: 'btn btn-primary'
+
 .autopagerize_page_element
-  - @maintenances.each do |maintenance|
-    %p= link_to maintenance.name, maintenance
-= paginate @maintenances
+  %table.table.table-hover
+    %thead
+      %tr
+        %th{scope: 'col'}= Maintenance.model_name.human
+        %th{scope: 'col'}= Car.model_name.human
+        %th{scope: 'col', colspan: '2'}
+
+      - @maintenances.each do |maintenance|
```

Comment:
You can add a partial in this place
```
= render @maintenances
```
and add partial named '_maintenance.html.haml' with the contain:
```
%tr
         %td= link_to maintenance.name, maintenance
          %td= maintenance.car.decorate.name
          %td= link_to t('.edit'), edit_maintenance_path(maintenance)
          %td= link_to t('.destroy'), maintenance, method: :delete, data: { confirm: 'Are you sure?' }
```